tests/test_rewriter.py

```diff
@@ -147,7 +147,6 @@ def test_inline_block_at_end():
 "builtin.module"() ({
   %0 = "test.op"() : () -> !test.type<"int">
   "test.op"() ({
-  ^0:
```

Comment:
inlined source blocks are now consistently deleted when inlined into another block

Reply:
Makes sense to me, that's actually what MLIR does as well!